src/custom/components/Version/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import styled from 'styled-components'
+import { useActiveWeb3React } from 'hooks'
+import { ExternalLink, TYPE } from 'theme'
+import { GP_SETTLEMENT_CONTRACT_ADDRESS } from 'custom/constants'
+import { getEtherscanLink } from 'utils'
+
+import { version as WEB, dependencies } from 'src/../../package.json'
+
+const VERSIONS = {
+  WEB,
+  CONTRACTS: dependencies['@gnosis.pm/gp-v2-contracts']
+}
```

Comment:
Better do
```ts
import { version: contractsVersion } from '@gnosis.pm/gp-v2-contracts/package.json'
```
to get exact version, not range `^0.0.1-alpha.8`

Reply:
good idea